go-controller/pkg/node/ovn_test.go

```diff
@@ -10,15 +10,20 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/kubernetes/fake"
 	"k8s.io/client-go/tools/record"
+
+	egressfirewallfake "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/crd/egressfirewall/v1/apis/clientset/versioned/fake"
+	apiextensionsfake "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/fake"
 )
 
 type FakeOVNNode struct {
-	node       *OvnNode
-	watcher    *factory.WatchFactory
-	stopChan   chan struct{}
-	recorder   *record.FakeRecorder
-	fakeClient *fake.Clientset
-	fakeExec   *ovntest.FakeExec
+	node             *OvnNode
+	watcher          *factory.WatchFactory
+	stopChan         chan struct{}
```

Comment:
do you need a separate `stopChan` for `egressfirewallwatchfactory`?

Reply:
at this point the node doesn't actually do anything with egressFirewall so I would say no